### DISCUSSION TITLE
Timestamp preservation when uploading to Wasabi S3

### DISCUSSION THREAD

**Discussion by Calum-Quinn** [iterate-ch/cyberduck#15487] [Q&A]
Hello,
I am currently trying to upload files from a file server into a Cryptomator encrypted vault from Wasabi.
I have ticked the "Preserve modification date" under transfers/uploads.
Once I then access the files in Wasabi after the upload all the timestamps (modification,access and creation) have been set to the date and time of the upload.

I have also tried using the Timestamp connection profile and this made no difference.

Anyone know why this might be happening?

**Comment by tatze96** [comment]
Timestamp preservation for the protocol S3 only works while using the S3 (Timestamps) connection profile. Note that the timestamps will only be preserved and displayed correctly for items uploaded to the server through the timestamp connection profile.

For more information, refer to our [wiki page](https://docs.cyberduck.io/protocols/s3/#modification-date).

**Reply by Calum-Quinn** [reply]
Thanks for your reply but as mentioned in the original post, I tried using the S3 timestamps connection profile and this made no difference and the timestamps were not preserved.
